server/EAP7-1075_read-provisioning-config.adoc

```diff
@@ -0,0 +1,57 @@
+= EAP7-1075 read-provisioning-config management operation
```

Comment:
Please rename and retitle this to use the WFCORE.  EAP7 JIRAs are secondary paperwork.